mentoring-wg/strategy.md

```diff
@@ -0,0 +1,33 @@
+CNCF TAG Contributor Strategy
```

Comment:
This is a lot of duplicate info with the FAQs.md page. Just a heads up that it's going to cause out of date / conflicting info over time as people will not realize they need to update both. Maybe just link to strategy here from the FAQ page and not repeat it?

Reply:
I agree -- it looks like  we completely dup'd the page. Will remove FAQs for now till we need it.